wiring/src/spark_wiring_ble.cpp

```diff
@@ -1143,22 +1143,22 @@ ssize_t BleCharacteristic::setValue(const uint8_t* buf, size_t len, BleTxRxType
     if (impl()->local()) {
         int ret = SYSTEM_ERROR_NOT_SUPPORTED;
         // Updates the local characteristic value for peer to read.
-        if (impl()->properties() & BleCharacteristicProperty::READ) {
+        if (impl()->properties().isSet(BleCharacteristicProperty::READ)) {
```

Comment:
What's wrong with `operator&`? :)

Reply:
It returns `EnumFlags`, while we've removed the `bool()` operator, so it will result compilation error.